src/library/trackcollection.h

```diff
@@ -84,7 +84,7 @@ class TrackCollection : public QObject,
     void exportTrackMetadata(Track* pTrack) const;
 
     // Must be called from the main thread
-    void saveTrack(Track* pTrack);
+    Q_INVOKABLE void saveTrack(Track* pTrack);
```

Comment:
Is there a difference between Q_INVOKABLE and the "private slots:" region? 
I would prefer the later, because it is used elsewhere.